src/client/components/threatmodal/threatmodal.js

```diff
@@ -103,35 +98,34 @@ class ThreatModal extends React.Component {
+              <Input type="textarea" name="description" id="description" disabled={!this.isOwner} style={{ height: 150 }} value={this.state.description} onBlur={(e) => this.props.moves.updateThreat("description", e.target.value)} onChange={(e) => this.setState({description: e.target.value})} />
             </FormGroup>
             <FormGroup hidden={this.props.G.threat.owner !== this.props.playerID}>
               <div className="checkbox-item">
-                <Input className="pointer" type="checkbox" id="showMitigation" onChange={(e) => this.toggleMitigationForm(e.target.checked)} />
+                <Input className="pointer" type="checkbox" id="showMitigation" onChange={(e) => this.toggleMitigationField(e.target.checked)} />
                 <Label for="showMitigation">Add a mitigation <em>(optional)</em></Label>
               </div>
             </FormGroup>
-            <FormGroup hidden={this.props.G.threat.owner === this.props.playerID && !this.state.showMigitation}>
+            <FormGroup hidden={this.props.G.threat.owner === this.props.playerID && !this.state.showMitigation}>
               <Label for="mitigation">Mitigation</Label>
-              <Input type="textarea" name="mitigation" id="mitigation" disabled={this.props.G.threat.owner !== this.props.playerID} style={{ height: 150 }} value={this.state.mitigation} onBlur={(e) => this.props.moves.updateThreat("mitigation", e.target.value)} onChange={(e) => this.updateState("mitigation", e.target.value)} />
+              <Input type="textarea" name="mitigation" id="mitigation" disabled={!this.isOwner} style={{ height: 150 }} value={this.state.mitigation} onBlur={(e) => this.props.moves.updateThreat("mitigation", e.target.value)} onChange={(e) => this.setState({mitigation: e.target.value})} />
             </FormGroup>
           </ModalBody>
-          <ModalFooter>
-            <Button color="success" className="mr-auto" hidden={this.props.G.threat.owner !== this.props.playerID} onClick={() => this.saveThreat()}>Save</Button>{' '}
-            <Button color="primary" disabled={this.isInvalid()} onClick={() => this.addOrUpdate()}>{(this.props.G.threat.new ? 'Save & Add' : 'Save & Update')}</Button>{' '}
-            <Button color="secondary" disabled={this.props.G.threat.owner !== this.props.playerID} onClick={() => this.props.moves.toggleModal()}>Cancel</Button>
-          </ModalFooter>
-          <ModalFooter hidden={this.props.G.threat.owner !== this.props.playerID}>
-            <small className="mr-auto text-muted"><b>TIP:</b> Saving would allow other players to view your changes instantly.</small>
-          </ModalFooter>
+          {
+            this.isOwner &&
```

Comment:
A general question: Is it better to use the `hidden` parameter or conditional rendering for these reactstrap components? At the moment, both is used in this file...
We should make it consistent.

Reply:
I think I would prefer to use conditional rendering. If they are just hidden, they will still be in the DOM, which doesn't make a lot of sense, IMO.